icepyx/core/granules.py

```diff
@@ -7,12 +9,15 @@
 import numpy as np
 import os
 import pprint
-from xml.etree import ElementTree as ET
 import zipfile
+from typing import Final
+from requests.compat import unquote
+from xml.etree import ElementTree as ET
 
 import icepyx.core.APIformatting as apifmt
-from icepyx.core.auth import EarthdataAuthMixin
 import icepyx.core.exceptions
+from icepyx.core.auth import EarthdataAuthMixin
```

Comment:
Why is this reordering happening? I thought we'd addressed it (though I can't quickly find in which ruff/reformatting PR).

Reply:
I thought so too. :thinking:

Reply:
Rebased, this should be resolved.